### PR TITLE
fix gui.py

### PR DESCRIPTION
**Pull request by JackEllie**
Fixed some naming issues.

### CODE REVIEW

File: gui.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ def __init__(
         初始化
         """
         try:
+            self.config = MyConfig()
```
Comment (Tps-F) [inline gui.py:51]: I don't think you need to put config in self.

Comment (JackEllie) [reply gui.py:51]: If not done so, the following error will be encountered. Perhaps you have better suggestions for modifications. This part is up to you to decide.

```
*cpt["config"], is_half=MyConfig().is_half
NameError: name 'config' is not defined
```

Comment (Tps-F) [reply gui.py:51]: No, you're right.
It have a class with the same name.
Thank you for the correction.

**Comment by Tps-F** [comment]
Once I check the operation and if it is working, I will merge it.

File: gui.py (hunk 1 above)
Comment (fumiama) [reply gui.py:51]: Well, why we have to rename `Config` class to `MyConfig` ? 😂

Comment (Tps-F) [reply gui.py:51]: I think that would be best.

Comment (Tps-F) [reply gui.py:51]: @fumiama Shall I do it?

Comment (fumiama) [reply gui.py:51]: > @fumiama Shall I do it?

Sure. You can edit this PR directly.

Comment (Tps-F) [reply gui.py:51]: @fumiama
I made some changes that are not included in this PR, so I created another PR #340
I'd like to see #340 merged after this PR is merged

**Review by Tps-F — APPROVED** [review]

**Pull request merged**